docs/source/guide/tasks.md

```diff
@@ -102,6 +103,116 @@ http://example.com/text.txt
 
 {% enddetails %}
 
+
+## How to retrieve data
+
+There are several steps to retrieve the data to display in the `Object` tag. The data retrieval is also used in [dynamic choices](/templates/serp_ranking.html) and [labels](/templates/inventory_tracking.html). Use the following parameters in the `Object` tag.
+
+### `value` (required)
+
+The `value` parameter represents the source of the data. It can be plain text or a step of complex data retrieval system. It can be denoted using the following forms:
+`value` (required)
+
+#### Variables 
```

Comment:
all subsections should be sharable via anchor links

Reply:
We already have too much levels, anchors are missed from 4+ level. We can move this Data retrieval part to a separate page